etc/scripts/install.sh

```diff
@@ -44,7 +44,7 @@ enable_interaction_with_registry
 # skip tag resolving for internal registry
 # OpenShift 3 and 4 place the registry in different locations, hence
 # the two hostnames here
-$CMD -n knative-serving get cm config-controller -oyaml | sed "s/\(^ *registriesSkippingTagResolving.*$\)/\1,docker-registry.default.svc:5000,image-registry.openshift-image-registry.svc:5000/" | oc apply -f -
+$CMD -n knative-serving get cm config-controller -oyaml | yq w - "data.registriesSkippingTagResolving" "ko.local,dev.local,docker-registry.default.svc:5000,image-registry.openshift-image-registry.svc:5000" | $CMD apply -f -
```

Comment:
I don't love the assumption of `yq` here, because as long as we're citing this repo as the way to install the dev preview, we'll need to update docs. I'd prefer a `sed` solution, if possible.

Reply:
sure, let me give it a try with `sed`

Reply:
Thanks! And fwiw, though I love that you made it backwards-compatible, I'm not sure it needs to be, since master tracks the latest upstream, so if that makes using `sed` simpler...